docs/configuration.rst

```diff
@@ -360,3 +344,28 @@ You'll remain logged in to ``oidcprovider``, and the account will persist until
 the ``oidcprovider`` container is stopped.
 You can visit http://oidc.127.0.0.1.nip.io:8081/account/logout to manually log
 out.
+
+
+Eliot configuration
+===================
+
+Webapp
+------
+
+The Eliot webapp is run as worker processes by Gunicorn which is run by Circus.
+
+.. autocomponent:: eliot.app.AppConfig
+   :hide-classname:
+   :namespace: eliot
+   :case: upper
+
+
+Disk cache manager
+------------------
+
+The Eliot disk cache manager is run as a single process by Circus.
+
+.. autocomponent:: eliot.cache_manager.AppConfig
+   :hide-classname:
+   :namespace: eliot
+   :case: upper
```

Comment:
It's super easy to document Eliot configuration because it uses Everett and Everett has a Sphinx extension.